src/components/bookappointment/modal/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { Close } from '@mui/icons-material';
+import {
+  CssBaseline,
+  Container, Paper, IconButton, Typography, Stepper, Step, StepLabel, Box, Button, Modal,
+} from '@mui/material';
+import * as React from 'react';
+import { FormikProps, useFormik } from 'formik';
+import ModelStyle from '../../../pages/therapistProfile/classes';

```

Comment:
too long path for import 
please import it from the index/create the file in the same folder